components/Snackbar/src/MDCSnackbarMessage.h

```diff
@@ -169,6 +170,13 @@ extern NSString *__nonnull const MDCSnackbarMessageBoldAttributeName;
  */
 @property(nonatomic) BOOL focusOnShow;
 
+/**
+ A block that is invoked when the corresponding MDCSnackbarMessageView of the MDCSnackbarMessage
+ instance will present itself.
```

Comment:
Nit: the `MDCSnackbarMessageView` does not present itself, it is presented by the `MDCSnackbarManager`. Maybe 

> A block that is invoked when the corresponding @c MDCSnackbarMessageView of the @c MDCSnackbarMessage instance will be presented. Use this to customize @c MDCSnackbarMessageView before presentation.

instead

Reply:
love it!